hypervisor/Makefile

```diff
@@ -227,6 +226,34 @@ S_OBJS := $(patsubst %.S,$(HV_OBJDIR)/%.o,$(S_SRCS))
 DISTCLEAN_OBJS := $(shell find $(BASEDIR) -name '*.o')
 VERSION := $(HV_OBJDIR)/include/version.h
 
+# Create platform_acpi_info.h
+SOURCE_ACPI_INFO_HEADER := bsp/include/$(CONFIG_BOARD)_acpi_info.h
+TEMPLATE_ACPI_INFO_HEADER := bsp/include/$(CONFIG_PLATFORM)/platform_acpi_info.h
+TARGET_ACPI_INFO_HEADER := $(HV_OBJDIR)/include/platform_acpi_info.h
+
+$(TARGET_ACPI_INFO_HEADER): $(HV_OBJDIR)/$(HV_CONFIG)
+ifeq (,$(wildcard $(SOURCE_ACPI_INFO_HEADER)))
+	@echo "******* No ACPI info found *******" && \
+	 echo "Expect ACPI info header at $(SOURCE_ACPI_INFO_HEADER)" && \
+	 echo "" && \
+	 echo "The ACPI info header for this board is not available. Please use" && \
```

Comment:
We assume users will generate the ACPI info header using a tool (which will be released soon) on the target board. This is to be used along with CONSTANT_ACPI=y so that runtime ACPI parsing can be avoided to enhance boot time as well as reduce safety-related code. Currently CONSTANT_ACPI defaults to n, indicating that the hypervisor will parse the ACPI tables at boot time and fill in the ACPI info template.

We may not add such headers for different boards because the ACPI info also depends on BIOS, which can be different on the same board after a BIOS update.

@jsun26intel please correct me if I misunderstood anything. Thanks.